src/components/errors/index.js

```diff
@@ -1 +1 @@
-export { default } from './error-404'; // eslint-disable-line
+import { Error404 } from '.'; // eslint-disable-line
```

Comment:
We don't want to disable eslint completely on this line. There's a way to disable the one rule that we need to on this line

Reply:
Fixed it.